pkg/protocol/attestations.go

```diff
@@ -194,7 +194,7 @@ func (a *Attestations) processResponse(commitment *model.Commitment, attestation
 			}
 
 			return actualWeight
-		}) > 0 {
```

Comment:
What exactly does this change do?

Reply:
Compute returns the previous value so currently we only set something to be attested after we received the 2nd attestations.

I rewrote the logic to make it more clear that we now set sth to be attested only the first time we receive attestations

Reply:
Ok sounds good!